packages/gatsby-source-contentful/src/gatsby-node.ts

```diff
@@ -47,6 +48,17 @@ export const onPreInit: GatsbyNode["onPreInit"] = async (
   { store, reporter, actions },
   pluginOptions
 ) => {
+  // gatsby version is too old
+  if (!hasFeature(`track-inline-object-opt-out`)) {
+    reporter.panic({
+      id: CODES.GatsbyPluginMissing,
+      context: {
+        // TODO update message to reflect the actual version with track-inline-object-opt-out support
+        sourceMessage: `Used gatsby version is too old and doesn't support required features. Please update to gatsby@>=5.X.0`,
```

Comment:
This will need to be updated before the release once it's clear that Gatsby Version that will contain the addition.

For now it will mean that users will have to use `gatsby@ctf-next` (once new wave of canary is released that include this change) - so maybe temporarily we can use that version there whatever that will end up being?